translations/nl.json

```diff
@@ -0,0 +1,251 @@
+    "IGNORE_ROLE_ADDED": "✅ Negeer rol succesvol toegevoegd",
+    "IGNORE_ROLE_REMOVED" :"✅ Negeer rol succesvol verwijderd",
+    "BLOCK_WORDS_REMOVED": "✅ Geblokkeerde woorden succesvol verwijderd",
+    "BLOCK_WORDS_REGISTERED": "✅ Geblokkeerde woorden succesvol geregistreerd",
+    "CLEARED_LIST": "Je hebt je lijst met geblokkeerde woorden gewist.",
+    "FILTER_ON": "Je filter was ingeschakeld toen je dit command uitvoerde. Meon kan niet functioneren zonder een lijst met geblokkeerde woorden, dus de functie is uitgeschakeld.",
+    "NO_LONGER_ALLOWED_WORDS": "De woord(en) ||{}|| zijn niet langer toegestaan op je server.",
+    "BLOCKED_WORDS_NOTE": "Houd er rekening mee dat de antivloek-filter nog steeds is uitgeschakeld. Schakel het in met `{}antiswear toggle` om je server te beschermen.",
+    "PROFANITY_ACTIVATED": "✅ Antivloek-filter geactiveerd",
+    "PROFANITY_DEACTIVATED": "✅ Antivloek-filter uitgeschakeld",
+    "NOW_SAFE_FROM_BAD_WORDS": "Je server is nu veilig voor slechte woorden.",
+    "NO_LONGER_SAFE_FROM_BAD_WORDS": "Je server is nu niet meer veilig voor slechte woorden.",
+    "NOT_SET_UP_BAD_WORDS": "Je hebt geen woorden ingesteld om te blokkeren! Gebruik het onderstaande command om een aantal woorden toe te voegen!\n`{ctx.prefix}antiswear block <word>, [word], [word], ...`",
+    "TOGGLE_HELP": "Toggle Help",
+    "LIST_TOGGLE": "Hier is een lijst met instellingen die je kunt in- of uitschakelen.",
+    "TOGGLE_HELP_COMMANDNOTFOUND": "Geeft een error weer als het command die je zocht niet bestaat.",
+    "CURRENTLY": "Huidig",
+    "TOGGLE_HELP_PREFIXERROR": "Geeft een error weer als een lid probeert de prefix `{}` te gebruiken terwijl de werkelijk prefix anders is. Het wordt aanbevolen om dit uit te schakelen als je meer dan één bot heeft met de prefix `{}`.",
+    "CONFIGURE_ANTI_PROFANITY": "Configuratie Antivloek-Filter",
+    "ANTI_PROFANITY_DESC": "De antivloek-filter van meon zal alle slechte woorden die leden versturen scannen en verwijderen!",
+    "ANTI_PROFANITY_PERM": "Leden met de `BEHEERDER` toestemming zullen altijd genegeerd worden. Je mag ook een moderator rol instellen die ook wordt genegeerd.",
+    "ANTI_PROFANITY_TOGGLE": "Schakel deze feature aan of uit met `{}antiswear toggle on/off`",
+    "ANTI_PROFANITY_CURRENTLY": "Deze feature is momenteel **{}**.",
+    "ON": "AAN",
+    "OFF": "UIT",
+    "BLOCKED_WORDS": "Geblokkeerde Woorden",
+    "NOT_SET": "Niet ingesteld",
+    "SET_BLOCKED_WORDS": "*Stel geblokeerde woorden in met `{}antiswear block <word>, [word], [word] ...` en reset ze met `{}antiswear block disable`.*",
+    "LOG_CHANNEL": "Log Kanaal",
+    "SET_LOG_CHANNEL": "*Stel het log kanaal in met `{}antiswear log <channel>` en reset het met `{}antiswear log disable`.*",
+    "IGNORE_ROLE": "Genegeerde Rol",
+    "SET_IGNORE_ROLE": "*Stel de genegeerde rol in met `{}antiswear ignore <role>` en reset ze met `{}antiswear ignore disable`.*"
+  }

```

Comment:
The [MESSAGE_CONTENT_EMPTY](https://github.com/BoupsInc/meon-translations/blob/master/translations/en.json#L251) key is missing